server/utils/scraper.ts

```diff
@@ -269,6 +269,7 @@ let formatSectionInfo = async (elmHandle: ElementHandle, browser: Browser): Prom
  * @param {Page} page The Page to be configured
  */
 let pageConfig = async (page: Page) => {
+    await page.setDefaultNavigationTimeout(0);
```

Comment:
What happens if it times out? Would we get any notifications or do we just do a quick sanity check after?

Reply:
So... I think some of the pages on SSC take a long time to scrape. Before I added this line, it kept timing out after 30000ms (which is default). This just lets the scraper keep going. 

Also, there are a few additional issues - which will probably force us to do manual scrapes instead of scheduled ones for now. Firstly, after testing it out a bit, scheduled scrapes with specific dates and times don't seem to work. I think it's been reported in node-cron https://github.com/node-cron/node-cron/issues/261 recently. In fact, we should probably get comment out the scheduled scraper for now.

Secondly, our db setup currently uses courses from the output json file. I realized that even if the scheduled scraper were working, I don't think our output json file actually gets updated (ie: heroku can't push changes to our git repo to update the output json) and I don't think Heroku can update the code base by itself (@resurreccionl can you confirm if this logic is correct?)

In short:
1) `await page.setDefaultNavigationTimeout(0)` fixes local time outs when scraping
2) node-cron seems to be broken for schedule scrapes
3) Our db setup relies on output.json changing (this is a code change), I don't think Heroku can update our code base

Reply:
yikes, i wasn't aware of it until you mentioned the possibility, but yes heroku doesn't persist changes to the filesystem because of how constantly dynos restart https://devcenter.heroku.com/articles/dynos#restarting
pretty disappointing that `node-cron` stopped working consistently too

Reply:
Yeah I didn't realize until now. It's okay though since manual scrapes are easy enough and node-cron should get fixed soon